requirements.txt

```diff
@@ -53,3 +53,7 @@ swifter==1.3.4
 telebot==0.0.5
 selenium==4.11.2
 apache-airflow-client==2.6.2
+openai==1.45.0
+langchain==0.2.16
+langchain_openai==0.1.25
+python-dotenv
```

Comment:
Is `python-dotenv` necessary?
Also, `SQLAlchemy==1.4` modification necessary?
(I think we should perform packages upgrade, but it will be in a different pr with suitable tests)